sdk/eventhub/azure-eventhubs/azure/eventhub/configuration.py

```diff
@@ -8,7 +8,11 @@
 class _Configuration(object):
     def __init__(self, **kwargs):
         self.user_agent = kwargs.get("user_agent")
-        self.max_retries = kwargs.get("max_retries", 3)
+        self.retry_total = kwargs.pop('retry_total', 3)
+        self.max_retries = self.retry_total or kwargs.get("max_retries", 3)
```

Comment:
given retry_total has default value, kwargs.get("max_retries", 3) will never be used?

Reply:
good catch